app/Entities/Models/Page.php

```diff
@@ -126,4 +126,18 @@ public function forJsonDisplay(): Page
         $refreshed->html = (new PageContent($refreshed))->render();
         return $refreshed;
     }
+
+    public function getCoverImage(): string
+    {
+        $dom = new \DomDocument();
+        $dom->loadHTML($this->html);
+        $images = $dom->getElementsByTagName('img');
+
+        try {
+            $cover = $images->length > 0 ? $images[0]->getAttribute('src') : $this->book->getBookCover();
+        } catch (Exception $err) {
```

Comment:
We should either add a `use \Exception;` statement at the top or put `\Exception` here.

Reply:
@Abijeet See changes in #48587d2 and let me know your thoughts.